lib/generators/sufia/install_generator.rb

```diff
@@ -87,12 +87,6 @@ def inject_sufia_application_controller_behavior
       end
     end
 
-    def use_blacklight_layout_theme
-      file_path = "app/controllers/application_controller.rb"
-      return unless File.exist?(file_path)
-      gsub_file file_path, /with_themed_layout '1_column'/, "layout 'sufia-one-column'"
```

Comment:
can you add a note about this in https://github.com/projecthydra/sufia/issues/2491 ?